index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,400;0,700;1,100&family=Roboto:wght@100;400&display=swap" rel="stylesheet">
+    <title>My Portfolio</title>
+    <script src="https://kit.fontawesome.com/242e63f88a.js" crossorigin="anonymous"></script>    
+</head>
+<body>
+    <section id="section-1">
+        <div id="not-bar">
+            <p>9:41</p>
+            <div class="fas-container">
+                <i class="fas fa-signal"></i>
+                <i class="fas fa-wifi"></i>
+                <i class="fas fa-battery-full"></i>
+            </div>            
+        </div>
+        <header id="header">
+            <span class="my-logo">My logo</span>
```

Comment:
- [ ]  I strongly recommend to put `logo` into an `<a>` tag. On web pages, logos are usually links that take users to homepage, this would improve usability of your project! 👍🏻

Reply:
Thank you, excuse me can I implement this change and then merge to the main branch, or does it need to be pull requested again?

Reply:
Your project is approved, you can merge branches, there is no need to open a new pull request 😊

Reply:
Ok, @ShoiraTa Thank your time and your feedback, I really appreciate all the work the code reviewers do for us.

Reply:
Thanks :blush: @Jord4N-AC